community/detectors/apache_activemq_cve_2023_46604/src/test/java/com/google/tsunami/plugins/detectors/cves/cve202346604/Cve202346604DetectorTest.java

```diff
@@ -196,6 +231,60 @@ public void detect_whenNotVulnerable_returnsNoVulnerability() throws Exception {
     assertThat(detectionReports.getDetectionReportsList()).isEmpty();
   }
 
+  @Test
+  public void detect_whenVulnerable_returnsVulnerability_not_use_oob() throws Exception {
```

Comment:
Please follow the capitalization from Google's style guide and follow the correct test naming format: `detect_whenVulnerableWithoutOob_returnsVulnerability`